javascript/CodeChallenges/CC-16_find-maximum-binary-tree/find-maximum-binary-tree.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+findMaximumValue(){
+  let array = this.inOrder(this.root);
+  for (let i=0; i < array.length; i++){
+    var max = 0;
+    if (array[i] > max){
+      max = array[i];
+    }
+  }
+  return max;
+}
```

Comment:
I would really like to see this challenge added to your Binary Tree versus living in a separate file.